Pod/Classes/MBCircularProgressBarView.m

```diff
@@ -58,15 +58,14 @@ -(void)initView:(CGRect)frame{
     [self setProgressLineWidth:14.f];
     [self setProgressAngle:80.f];
     [self setUnitFontSize:-1];
-    [self setValueFontSize:-1];
     [self setValueDecimalFontSize:-1];
     [self setDecimalPlaces:0];
     [self setShowUnitString:YES];
     [self setShowValueString:YES];
-    [self setValueFontName:@"HelveticaNeue-Thin"];
     [self setTextOffset:CGPointMake(0, 0)];
     [self setUnitFontName:@"HelveticaNeue-Thin"];
     [self setCountdown:NO];
+    [self setValueFont:[UIFont boldSystemFontOfSize:60]];
```

Comment:
If I understand well, this is the place we set our font now? (we use the default one)
If that's the case, I think `Medium` font was used

Reply:
@squall2022 yes. here. But in Zeplin we have bold one: https://zpl.io/bznvqo4.

Reply:
@wojteklu Ah ok, in fact this is not the same in old Congrat: https://zpl.io/br7Yo72
But I guess it's better to keep just one to be consistent, so `bold` is OK 👍